deploy/db/sql/populate.sql

```diff
@@ -0,0 +1,10 @@
+-- Populate wallets table
+INSERT INTO notary.wallets (wallet, irohausername)
+VALUES ('0x00aa39d30f0d20ff03a22ccfc30b7efbfca597c2', 'user1@notary');
```

Comment:
Maybe it is better to store all these addresses in a separate config file?

Reply:
It is testing data, not configuration. These data is used for manual and integration testing.